app/com/m3/octoparts/aggregator/service/PartRequestService.scala

```diff
@@ -12,5 +12,9 @@ import scala.concurrent.ExecutionContext
 class PartRequestService(
   val repository: ConfigsRepository,
   val handlerFactory: HttpHandlerFactory,
-  implicit val zipkinService: ZipkinServiceLike)(implicit val executionContext: ExecutionContext)
-    extends PartRequestServiceBase
+  implicit val zipkinService: ZipkinServiceLike,
+  val proxyDefinitions: Map[String, String] = Map())(implicit val executionContext: ExecutionContext)
```

Comment:
Let's leave out the default argument. We don't instantiate one of these often and it makes it easier to spot bugs at compile time.